src/plotman/plotman.py

```diff
@@ -137,120 +137,121 @@ def main():
         #
-        elif args.cmd in [ 'details', 'files', 'kill', 'suspend', 'resume' ]:
-            print(args)
-
-            selected = []
-
-            # TODO: clean up treatment of wildcard
-            if args.idprefix[0] == 'all':
-                selected = jobs
-            else:
-                # TODO: allow multiple idprefixes, not just take the first
-                selected = manager.select_jobs_by_partial_id(jobs, args.idprefix[0])
-                if (len(selected) == 0):
-                    print('Error: %s matched no jobs.' % args.idprefix[0])
-                elif len(selected) > 1:
-                    print('Error: "%s" matched multiple jobs:' % args.idprefix[0])
-                    for j in selected:
-                        print('  %s' % j.plot_id)
-                    selected = []
-
-            for job in selected:
-                if args.cmd == 'details':
-                    print(job.status_str_long())
-
-                elif args.cmd == 'files':
-                    temp_files = job.get_temp_files()
-                    for f in temp_files:
-                        print('  %s' % f)
-
-                elif args.cmd == 'kill':
-                    # First suspend so job doesn't create new files
-                    print('Pausing PID %d, plot id %s' % (job.proc.pid, job.plot_id))
-                    job.suspend()
-
-                    temp_files = job.get_temp_files()
-                    print('Will kill pid %d, plot id %s' % (job.proc.pid, job.plot_id))
-                    print('Will delete %d temp files' % len(temp_files))
-                    conf = input('Are you sure? ("y" to confirm): ')
-                    if (conf != 'y'):
-                        print('canceled.  If you wish to resume the job, do so manually.')
-                    else:
-                        print('killing...')
-                        job.cancel()
-                        print('cleaning up temp files...')
+        elif args.cmd == 'analyze':
+
+            analyzer.analyze(args.logfile, args.clipterminals,
+                    args.bytmp, args.bybitfield)
+
+        else:
+            jobs = Job.get_running_jobs(cfg.directories.log)
+
+            # Status report
+            if args.cmd == 'status':
+                result = "{0}\n\n{1}\n\nUpdated at: {2}".format(
+                    reporting.status_report(jobs, get_term_width()),
+                    reporting.summary(jobs),
+                    datetime.datetime.today().strftime("%c"),
+                )
+                print(result)
+
+            # Directories report
+            elif args.cmd == 'dirs':
+                print(reporting.dirs_report(jobs, cfg.directories, cfg.scheduling, get_term_width()))
+
+            elif args.cmd == 'interactive':
+                interactive.run_interactive(cfg=cfg)
+
+            # Start running archival
+            elif args.cmd == 'archive':
+                print('...starting archive loop')
```

Comment:
Add a check to make sure we'll be able to actually write the logs before even starting: (related to another comment I made)

```python
if not path.exists(cfg.directories.log) or not os.access(cfg.directories.log, os.W_OK):
    print(f"Error: Directory '{cfg.directories.log}' doesn't exist or isn't writable")
    sys.exit(1)
```